libs/transloco/src/lib/transloco.providers.ts

```diff
@@ -76,12 +76,12 @@ export function provideTranslocoLoader(loader: Type<TranslocoLoader>) {
   ]);
 }
 
-export function provideTranslocoScope(scope: TranslocoScope) {
-  return {
+export function provideTranslocoScopes(...scopes: TranslocoScope[]) {
```

Comment:
keep the original name `provideTranslocoScope` and let's merge this, no need to make this a breaking change over a name as all the previous usages will still work as expected (correct my if I'm wrong)